README.md

```diff
@@ -206,8 +206,8 @@ _\* databases are optional, however bot memory will not persist between differen
 
 ### Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](/docs/contributing.md)].
-<a href="graphs/contributors"><img src="https://opencollective.com/opsdroid/contributors.svg?width=890" /></a>
+This project exists thanks to all the people who contribute. [[Contribute](contributing.md)].
```

Comment:
This one is tricky because it will now be broken from the GitHub README.

Maybe this needs to become an absolute URL?

Reply:
Interesting. Which one is priority? github or website? (i mean this is what we can get if we use the index as a link to the readme). On a website, you getting 404 too.

Reply:
Both should work. Perhaps adding a symlink from `contributing.md` to `docs/contributing.md` would be a suitable fix?

Reply:
Honestly? It looks like a bad design. You will get one more 404 (this time in GitHub, for tutorials/introduction.md that not exists).

Reply:
> Honestly? It looks like a bad design. 
Why?

Reply:
It introduces one more link that gives 404. Give me a few minutes, need to think about the issue.

Reply:
Why would it? Just leave the file in docs/contributing.md and update the link as you suggested so the website works fine. Then add a symlink in the repo from the root so that the repo README works.

Reply:
Well, lets try it. ping me, I will check site online once it will be regenerated.

Reply:
![download](https://user-images.githubusercontent.com/651824/66063228-a1b7dc80-e54b-11e9-807c-db30c0ecbd98.gif)

You know what? You have solve this issue already. Checkout the **To get started see the...*** at readme.md or index page of the https://docs.opsdroid.dev/en/stable/ or of the https://opsdroid.readthedocs.io/en/latest/

Reply:
Sure let's to that!